litestar/security/jwt/token.py

```diff
@@ -85,8 +90,35 @@ def decode(cls, encoded_token: str, secret: str | dict[str, str], algorithm: str
         Raises:
             NotAuthorizedException: If the token is invalid.
         """
+
+        def base64url_decode(code: str) -> bytes:
+            padding = "=" * (4 - (len(code) % 4))
+            return base64.urlsafe_b64decode(code + padding)
```

Comment:
Why are we doing custom b64 decoding here?

Reply:
This also looks like functionality from: https://github.com/mpdavis/python-jose/blob/master/jose/utils.py#L65